pkg/common/events/states.go

```diff
@@ -93,6 +95,7 @@ func States() *AllStates {
 				Killing:    "Killing",
 				Killed:     "Killed",
 				Failed:     "Failed",
```

Comment:
Is not related to this change but can you please add the string representation for Failing state? `Failing:    "Failing",`